src/packages.builds

```diff
@@ -8,14 +8,37 @@
     <BuildAllOSGroups Condition="'$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
   </PropertyGroup>
 
+  <!--
+    the following packages are built and shipped from TFS, so while we still want
+    to build them we don't want to ship them.  by setting NonShippingPackage to true
+    the packages will still be built but placed in a different location.
+  -->
   <ItemGroup>
-    <Project Include="Native\pkg\**\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+    <NonShippingPackage Include="Native\pkg\runtime.native.System.Data.SqlClient.sni\*.builds" Condition="'$(SkipNativePackageBuild)' != 'true'">
+      <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true;NonShippingPackage=true</AdditionalProperties>
```

Comment:
Seems like there is a lot of duplicated meta-data that we should be able to eliminate. Would it be easier if we created a NativeNonShippingPackage and a NonShippingPackage so we can easily add the NonShippingPackage=true to the full set and then we can get all the other metadata below when you add them via the two sets.